cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java

```diff
@@ -63,6 +65,12 @@ public String getDisplayName() {
         public String getBranchName() {
             return branchName;
```

Comment:
AFAIK this method is used only from `ParallelStepTest.shouldHaveParallelStepsInTheOrder`, so it could be deleted in favor of `getThreadName`.